starlette/concurrency.py

```diff
@@ -11,8 +12,15 @@
 T = typing.TypeVar("T")
 
 
+def _create_task(coro: Coroutine) -> asyncio.Task:
```

Comment:
This is not going to pass the coverage requirements 😞 not sure what to do about that?

Reply:
Technically, this `create_task` change is not necessary to fix the bug and I can revert this commit if wanted.

See this for why the change: https://docs.python.org/3/library/asyncio-task.html#asyncio-example-wait-coroutine

Reply:
Oh I see #889 attempted to tackle this too, with a slightly different approach.

Reply:
I've removed this change from this PR as it's technically not required.